from src.application.interfaces.exceptions.exception_messages_interface import ExceptionMessagesInterface

class ExceptionMessages(ExceptionMessagesInterface):
  INTERNAL_SERVER_ERROR = 'Internal server error. Try again!'
  USER_NOT_FOUND = 'User not founded'
